src/components/keypad/button.js

```diff
@@ -6,44 +7,45 @@ import Color from "@khanacademy/wonder-blocks-color";
 
 import Clickable from "@khanacademy/wonder-blocks-clickable";
 
-type Props = {
-    onPress: () => void,
-    children: mixed,
-};
+import type {Node} from "React";
+import type {StyleType} from "@khanacademy/wonder-blocks-core";
 
 const styles = StyleSheet.create({
     base: {
-        width: 44,
-        height: 44,
+        display: "flex",
```

Comment:
FYI: `View` has `display: "flex"` set by default.  See https://github.com/Khan/wonder-blocks/blob/master/packages/wonder-blocks-core/components/view.js#L9-L26 for the full list.